R/plotPrevalence.R

```diff
@@ -274,7 +275,7 @@ setMethod("plotPrevalentAbundance", signature = c(x = "SummarizedExperiment"),
                       panel.background=element_blank(),
                 ) +
                 # Create facets
-                facet_wrap(~as.factor(rowData(x)[[facet_by]])) 
+                facet_wrap(~as.factor(rowData(x)[["Kingdom"]])) 
```

Comment:
I think this should be `facet_wrap(vars(!!sym("facet_by")))`. Without testing I am not 100% sure, that the quosure stuff is correct. However, it is necessary since `facet_wrap(. ~ facet_by)` will/should get flagged by R CMD check due to the variable not being defined.

btw: the theme_call should not be necessary, since a theme is already set. I you need to tweak these settings, the functionality should be moved into `.prevalence_plotter`

Reply:
Ok, thanks. I forgot that "kingdom" there, because it wasn't working after those modifications that I made and I tried to solve why. But now it is working, because I used your code. 

Well, I don't think it is necessary to have those theme modifications there. Maybe best to have one kind of plot from one function, i don't know... I deleted those